test/run

```diff
@@ -253,23 +253,13 @@ testYarn() {
   assertCapturedSuccess
 }
 
-testYarnCacheDirectory() {
```

Comment:
Most of this test didn't make sense -- it was running assuming the feature flag was disabled, but the flag was at 100%, so it's not a real user scenario.